rewriter/src/main/java/com/facebook/coresql/rewriter/OrderByRewriter.java

```diff
@@ -35,31 +35,30 @@
 public class OrderByRewriter
         extends Rewriter
 {
-    PatternMatcher<Set<AstNode>> matcher;
     public Set<AstNode> patternMatchedNodes;
     private static final String REWRITE_NAME = "ORDER BY without LIMIT";
 
     public OrderByRewriter()
     {
-        this.matcher = new OrderByPatternMatcher();
         this.patternMatchedNodes = new HashSet<>();
     }
 
     @Override
-    public boolean rewritePatternIsPresent(String sql)
+    public boolean hasRewritePattern(AstNode root)
```

Comment:
We can remove this from the API and make the method private

Reply:
Made private.

Done